recipe/patches/0001-Use-a-folder-inside-lib-for-bitcode-instead-of-top-l.patch

```diff
@@ -1,18 +1,18 @@
-From 0a9363723a3f466fdb753a299b15feecced54f22 Mon Sep 17 00:00:00 2001
-From: Isuru Fernando <isuruf@gmail.com>
-Date: Tue, 5 Mar 2024 14:38:17 -0600
+From e52e01e13939b0a9235c2be6eebd44b2c5889b15 Mon Sep 17 00:00:00 2001
+From: Klaus Zimmermann <klaus.zimmermann@quansight.com>
```

Comment:
?

Reply:
Sorry, let me fix that (I did a new git format-patch locally).